crates/biome_js_analyze/tests/specs/nursery/noRestrictedImports/invalid.js

```diff
@@ -1,2 +1,3 @@
 import eslint from 'eslint';
+import ComponentA from 'antd/componentA';
 const l = require('lodash');
```

Comment:
In order to reduce the clutter of the diff, I suggest you to add this new import at the end.